R/annotation.R

```diff
@@ -219,15 +219,15 @@ annotate_driver_ssm = function(maf_df,
     return()
   }else{
     #this all assumes the user has left the noncoding variants in their MAF
-    kept_ssm = maf_df %>% 
+    kept_ssm = maf_df %>%
       dplyr::filter(Variant_Classification %in% coding_vc) %>%
       dplyr::filter(Hugo_Symbol %in% driver_genes)
 
     for(gene in names(include_noncoding)){
       message(paste("adding", unname(include_noncoding[gene]), "for", gene))
-      nc_ssm = maf_df %>% 
```

Comment:
Are all of these edits related to the feature you told us about last week, where Rstudio automatically cleans up white stray spaces etc? Handy!